test/async.js

```diff
@@ -0,0 +1,220 @@
+import crypto from 'crypto';
+import fs from 'fs';
+import path from 'path';
+
+import bufferEquals from 'buffer-equals';
+import pify from 'pify';
+import rewire from 'rewire';
+import rimraf from 'rimraf';
+import test from 'ava';
+import uuid from 'uuid';
+
+import cpFile from '../';
+import {assertDateEqual} from './_assert';
+
+const fsP = pify(fs, Promise);
+const THREE_HUNDRED_KILO = (100 * 3 * 1024) + 1;
+
+test.before(() => {
+	process.chdir(path.dirname(__dirname));
+});
+
+test.beforeEach(t => {
+	const src = t.context.src = uuid.v4();
+	const dest = t.context.dest = uuid.v4();
+	t.context.creates = [src,	dest];
```

Comment:
`t.context.creates = [src, dest];`, looks like you have a tab instead of a space there.